frontend/src/components/Login.jsx

```diff
@@ -33,6 +33,31 @@ const Login = () => {
 
       if (response.ok) {
         setUser(data); // Set the user in context with id and username
+
+        const maxAge = 1000 * 60 * 5;
```

Comment:
We should either store these values in consts, or perhaps they are already defined as consts within datetime library and we can re-use those